CONTRIBUTING.md

```diff
@@ -63,7 +63,7 @@ Nussknacker UI config with your model.
 #### Running full version of Designer from IntelliJ
 
 Before running from IDE you have to manually build:
-- run `npm ci && npm run build` in `ui/client` (only if you want to test/compile FE, see `Readme.md` in `ui/client` for more details)
+- run `npm ci && npm run build` in `ui/client` and in `ui/submodules` (only if you want to test/compile FE, see `Readme.md` in `ui/client` for more details)
```

Comment:
WDYT about just keep it in one place: "Build frontend according to [Building frontend](#building-frontend) instruction (only if you want ..." ?

Reply:
:+1: